projects/igniteui-angular/src/lib/grids/row.component.ts

```diff
@@ -240,7 +262,7 @@ export class IgxRowComponent<T extends IgxGridBaseComponent & IGridDataBindable>
     /**
      * @hidden
      */
-    protected defaultCssClass = 'igx-grid__tr';
+    public defaultCssClass = 'igx-grid__tr';
```

Comment:
I think this can stay `protected`

Reply:
Use inside the igxRowDrag directive and should be public

Reply:
`@internal` as well then. Also, doesn't the cloned element already have all the classes needed?